lesson02/launch_confg.tf

```diff
@@ -1,5 +1,5 @@
 variable "instance_type" {
-  default = "m4.large"
+  default = "t2.micro"
```

Comment:
Why not t3.micro or t3.small?

Reply:
Thought of the `t2` because they can be used out of the box in AWS but I'm not sure if `t3` have the same behavior.